src/DeVinci_frontend/components/SelectModelOption.svelte

```diff
@@ -205,14 +252,43 @@
       <label for={id} class="inline-flex items-center justify-between w-full h-full p-3 cursor-pointer peer-checked:border-solid peer-checked:cursor-default peer-checked:bg-[lightsteelblue] peer-checked:border-[#151b1e] peer-checked:text-[#151b1e] hover:text-gray-600 hover:bg-[lightsteelblue]">
         <div class="block">
           <div class="w-full text-[#151b1e] text-md font-semibold">{name}</div>
-          <div class="w-full text-sm font-normal">{parameters}</div>
+          <div class="w-full text-sm font-normal">
+            {parameters}
+          </div>
           <span class="performance-span text-[#151b1e] text-xs font-medium me-1.5 px-2.5 py-0.5 bg-gray-300 rounded border-2 border-[#151b1e]">{performance}</span>
-          <span class="bg-gray-100 text-gray-800 text-xs font-medium mx-0 px-2.5 py-0.5 rounded border border-gray-500">{size}</span>
-          <div class="inline-flex  bg-gray-100 mt-2 text-gray-800 text-xs font-medium mx-0 px-2 py-1 rounded border border-gray-500">
-            <div class="w-64">
-              {goodFor}
-            </div>
+          <span 
+            class="bg-gray-100 text-gray-800 text-xs font-medium mx-0 px-2.5 py-0.5 rounded border border-gray-500 relative inline-block cursor-pointer"
+            on:mouseenter={() => showVramTooltip = true}
+            on:mouseleave={() => showVramTooltip = false}
+          >
+            {vramRequired ? vramMBtoGB(vramRequired) : "N/A"}GB
+            {#if showVramTooltip}
+              <div class="absolute left-1/2 transform -translate-x-1/2 top-full mt-2 z-50 p-2 text-sm text-white bg-black rounded-lg shadow-lg whitespace-nowrap">
+                Aprox. model size to download
+              </div>
+            {/if}
+          </span>
+          
+          <!-- Language support -->
```

Comment:
we're indicating more than languages here (good of, incl. Math, Coding). How can we update this to reflect that?

Reply:
good point, updated with "Good for"